terraform/aws/main.tf

```diff
@@ -201,14 +204,15 @@ module "dataset_api" {
 }
 
 module "secor" {
-  source                  = "../modules/helm/secor"
-  env                     = var.env
-  building_block          = var.building_block
-  secor_sa_annotations    = "eks.amazonaws.com/role-arn: ${module.eks.secor_sa_iam_role}"
-  secor_chart_depends_on  = [module.kafka]
-  secor_namespace         = module.eks.secor_namespace
-  cloud_storage_bucket    = module.s3.s3_bucket
-  kubernetes_storage_class = var.kubernetes_storage_class
+  source                    = "../modules/helm/secor"
+  env                       = var.env
+  building_block            = var.building_block
+  secor_sa_annotations      = "eks.amazonaws.com/role-arn: ${module.eks.secor_sa_iam_role}"
+  secor_chart_depends_on    = [module.kafka]
+  secor_namespace           = module.eks.secor_namespace
+  cloud_storage_bucket      = module.s3.s3_bucket
+  kubernetes_storage_class  = var.kubernetes_storage_class
+  region                    = var.region
```

Comment:
are we updating the region value in the secor helm chart? since you are introducing the variable region variable in the terraform secor module

Reply:
If we are using helm chart we need to provide the region and then deploy. If we are using via terraform it will prompt for region.